rasa/shared/core/training_data/structures.py

```diff
@@ -131,6 +131,13 @@ def _user_string(story_step_element: UserUttered, e2e: bool) -> Text:
     def _bot_string(story_step_element: Event) -> Text:
         return f"    - {story_step_element.as_story_string()}\n"
 
+    @staticmethod
+    def _or_string(story_step_element: List[Event], e2e: bool) -> Text:
+        result = " OR ".join(
+            [element.as_story_string(e2e) for element in story_step_element]
```

Comment:
This should result in a type error as we don't know for sure that each event is a `UserUttered` event. We should check for that.